src/arrayMethodSort.js

```diff
@@ -4,9 +4,39 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareString) {
+    let count = 1;
+
+    while (count > 0) {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const currentValue = this[i];
+        const prevValue = this[i - 1];
+
+        if (compareFunction(prevValue, currentValue) > 0) {
+          this[i - 1] = currentValue;
+          this[i] = prevValue;
+          count++;
+        }
+      }
+    }
+
+    return this;
   };
 }
 
+const compareString = (prevValue, currentValue) => {
+  const strPrev = `${prevValue}`;
+  const strCurrent = `${currentValue}`;
+
+  if (strPrev > strCurrent) {
+    return 1;
+  } else if (strPrev < strCurrent) {
+    return -1;
+  } else {
+    return 0;
+  }
```

Comment:
You should avoid else if it is not necessary
```suggestion
  if (strPrev > strCurrent) {
    return 1;
  }  
  
  if (strPrev < strCurrent) {
    return -1;
  } 
    
  return 0;
```